### PR TITLE
Azure Lab Services 2021-11-15-preview mangement api minor version bump to 4.1.0

### PR DESCRIPTION
**Pull request by kthatipally**
# Contributing to the Azure SDK
This PR has the changes to get the new swagger spec with the below setting which makes the LabServicesClient constructors as public and bump up to a minor version from 4.0.0 to 4.1.0
# Bug Summary:
Customers were unable to create client to use AzureLabServices SDK as LabServicesClient doesn’t expose a public parameter that handles the ServiceClientCredentials. All the constructors exposed were internal and now has been fixed in this PR.
Azure Rest API Spec - Swagger Updated spec PR: https://github.com/Azure/azure-rest-api-specs/pull/19402

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by fengzhou-msft — DISMISSED** [review]

File: sdk/labservices/Microsoft.Azure.Management.LabServices/src/Microsoft.Azure.Management.LabServices.csproj [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
 
 	<PropertyGroup>
-		<Version>4.0.0</Version>
+		<Version>4.1.0</Version>
 		<Description>Microsoft.Azure.Management.LabServices</Description>
 		<AssemblyName>Microsoft.Azure.Management.LabServices</AssemblyName>
 		<PackageId>Microsoft.Azure.Management.LabServices</PackageId>
 		<PackageTags>LabServices;ManagedLabs;</PackageTags>
-		<PackageReleaseNotes>Release api version 2021-11-15-preview.</PackageReleaseNotes>
+		<PackageReleaseNotes>Minor Version Update - Bugs Fixed:LabServicesClient constructors are now public</PackageReleaseNotes>
```
Comment (ArthurMa1978) [inline sdk/labservices/Microsoft.Azure.Management.LabServices/src/Microsoft.Azure.Management.LabServices.csproj:12]: preview version of swagger spec can't be used to release a stable version of sdk

File: sdk/labservices/Microsoft.Azure.Management.LabServices/src/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -6,8 +6,8 @@
 
 [assembly: AssemblyTitle("Microsoft Azure Lab Services Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Lab Services Resources.")]
-[assembly: AssemblyVersion("4.0.0.0")]
-[assembly: AssemblyFileVersion("4.0.0.0")]
+[assembly: AssemblyVersion("4.1.0.0")]
```
Comment (ArthurMa1978) [inline sdk/labservices/Microsoft.Azure.Management.LabServices/src/Properties/AssemblyInfo.cs:9]: Not only this file, please follow https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec to update all

File: sdk/labservices/Microsoft.Azure.Management.LabServices/src/Microsoft.Azure.Management.LabServices.csproj (hunk 1 above)
Comment (kthatipally) [reply sdk/labservices/Microsoft.Azure.Management.LabServices/src/Microsoft.Azure.Management.LabServices.csproj:12]: Hi @ArthurMa1978 , Can you please share more details about the comment? We currently have only the preview version, we are yet to release the GA version. Thanks.

File: sdk/labservices/Microsoft.Azure.Management.LabServices/src/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (kthatipally) [reply sdk/labservices/Microsoft.Azure.Management.LabServices/src/Properties/AssemblyInfo.cs:9]: Hi @ArthurMa1978, Is it the Assembly version that needs an update here?

Comment (kthatipally) [reply sdk/labservices/Microsoft.Azure.Management.LabServices/src/Properties/AssemblyInfo.cs]: updated the Assembly version to beta version.

File: sdk/labservices/Microsoft.Azure.Management.LabServices/src/Microsoft.Azure.Management.LabServices.csproj [hunk 3]
```diff
@@ -4,13 +4,14 @@
 	<!-- Please do not move/edit code below this line -->
 
 	<PropertyGroup>
-		<Version>4.0.0</Version>
+		<Version>1.0.0-beta.1</Version>
```
Comment (ArthurMa1978) [inline sdk/labservices/Microsoft.Azure.Management.LabServices/src/Microsoft.Azure.Management.LabServices.csproj:7]: 4.1.0-beta.1

File: sdk/labservices/Microsoft.Azure.Management.LabServices/src/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -6,8 +6,8 @@
 
 [assembly: AssemblyTitle("Microsoft Azure Lab Services Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Lab Services Resources.")]
-[assembly: AssemblyVersion("4.0.0.0")]
-[assembly: AssemblyFileVersion("4.0.0.0")]
+[assembly: AssemblyVersion("1.0.0")]
```
Comment (ArthurMa1978) [inline sdk/labservices/Microsoft.Azure.Management.LabServices/src/Properties/AssemblyInfo.cs:9]: 4.1.0-beta.1

File: sdk/labservices/Microsoft.Azure.Management.LabServices/src/Microsoft.Azure.Management.LabServices.csproj [hunk 5]
```diff
@@ -4,13 +4,14 @@
 	<!-- Please do not move/edit code below this line -->
```
Comment (ArthurMa1978) [inline sdk/labservices/Microsoft.Azure.Management.LabServices/src/Microsoft.Azure.Management.LabServices.csproj:5]: Should have a CHANGELOG under this folder, like this https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventgrid/Microsoft.Azure.Management.EventGrid/CHANGELOG.md

File: sdk/labservices/Microsoft.Azure.Management.LabServices/src/Microsoft.Azure.Management.LabServices.csproj (hunk 3 above)
Comment (kthatipally) [reply sdk/labservices/Microsoft.Azure.Management.LabServices/src/Microsoft.Azure.Management.LabServices.csproj]: updated to 4.1.0-beta.1

File: sdk/labservices/Microsoft.Azure.Management.LabServices/src/Microsoft.Azure.Management.LabServices.csproj (hunk 5 above)
Comment (kthatipally) [reply sdk/labservices/Microsoft.Azure.Management.LabServices/src/Microsoft.Azure.Management.LabServices.csproj:5]: Added change.log file with the release information.

File: sdk/labservices/Microsoft.Azure.Management.LabServices/src/Properties/AssemblyInfo.cs (hunk 4 above)
Comment (kthatipally) [reply sdk/labservices/Microsoft.Azure.Management.LabServices/src/Properties/AssemblyInfo.cs]: updated the version to 4.1.0-beta.1

File: sdk/labservices/Microsoft.Azure.Management.LabServices/CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+# Release History
+
+## 4.1.0-preview (2022-07)
+- This release corresponds to the 2021-11-15-preview API version.
+-  Bugs Fixed:LabServicesClient constructors are now public
```
Comment (ArthurMa1978) [inline sdk/labservices/Microsoft.Azure.Management.LabServices/CHANGELOG.md:5]: One extra space should be removed

**Review by ArthurMa1978 — APPROVED** [review]

File: sdk/labservices/Microsoft.Azure.Management.LabServices/CHANGELOG.md (hunk 6 above)
Comment (kthatipally) [reply sdk/labservices/Microsoft.Azure.Management.LabServices/CHANGELOG.md]: Removed extra space in line number 5

**Pull request merged**